### ISSUE TITLE
A better way to mark praise as duplicates

### ISSUE THREAD

**Issue by kristoferlund** [CommonsBuild/praise#210]
## Details

@divine-comedian:

> I was braining some things this fine saturday from our quant review call yesterday - drafted a mock-up wireframe for a rework of marking duplicate praise. I dropped them in the miro board under the big header of Marking Duplicate Praise Revision
https://miro.com/app/board/o9J_lyTkWjY=/

> I'm suggesting we invert the process of marking duplicates in that instead of marking a praise as duplicate of another we simply highlight a praise then mark all the duplicates at once of that "original" praise. Adding in a nice header and some checkboxes. 

> We can remove the need to squint to find praise codes and typing them into a box - it can all be done with just a couple mouse clicks! 

## Wireframes

https://miro.com/app/board/o9J_lyTkWjY=/

## 💭 Who should be part of Advice Process on this issue

@divine-comedian @mattyg @Vyvy-vi @CFWHISPERER @nebs-dev

**Comment by kristoferlund** [comment]
From Discord:  
> Yes, I like it! I think we could do a similar flow for dismissing praise. The general functionality is: the ability to select on or more praise items and then perform an action that affects them all. Let's iterate on this one or two loops and flesh out the spec.

**Comment by kristoferlund** [comment]
I have simplified the flow slightly and together with @mattyg decided to move move the search/filter functionality to a separate issue to be implemented later.

## TL;DR
- A toolbar that stays pinned to top while scrolling
- Checkboxes next to praise items
- Select one or more to activate buttons and perform actions
- Dismiss button: Simple! Mark 3 praise, click, confirm, done.
- Mark as duplicate button: Mark at least 2 praise.
  - Dialog opens
  - The first (chronologically) of the marked praise becomes the "original". 
  - Give that praise a score
  - The others become duplicates of that one.

Wireframes: https://miro.com/app/board/o9J_lyTkWjY=/

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/9698363/165746760-66af1015-e1c4-463d-9df9-02cbce9e1a4c.png">

<img width="1283" alt="image" src="https://user-images.githubusercontent.com/9698363/165746823-3902f890-98b7-4bbb-954c-3cf66070a4cf.png">

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/9698363/165746910-c6908563-8a75-41b1-adf0-19a2152aff07.png">

**Comment by kristoferlund** [comment]
> One option would be to keep the previous UX 'mark as duplicate' button on each praise that opens the PraiseAutosuggest dialog. We could make the button visible only when hovering over a praise instance for a bit cleaner look.

Hmm, yeah. Let's try that. Keep the mark-as-duplicate button for now. 

Or.. context menu - right click 😂
